app/models/listing.rb

```diff
@@ -9,6 +9,10 @@ class Listing < ActiveRecord::Base
   belongs_to :category
   has_many :photos
   has_many :reservations
+  scope :city, -> (city) { where city: city }
+  scope :max_guests, -> (max_guests) { where("max_guests >= ?", max_guests) }
```

Comment:
Thanks! I still want to refine them so you can search by all criteria at the same time though.